core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -67,6 +69,11 @@ public static function getPageIdFromUrl()
 	{
 		@trigger_error('Using Frontend::getPageIdFromUrl() has been deprecated and will no longer work in Contao 5.0. Use the Symfony routing instead.', E_USER_DEPRECATED);
 
+		if (!System::getContainer()->getParameter('contao.legacy_routing'))
+		{
+			throw new LegacyRoutingException('Frontend::getPageIdFromUrl() requires legacy routing. Configure "prepend_locale" or "url_suffix" in the Contao bundle.');
```

Comment:
```suggestion
			throw new LegacyRoutingException('Frontend::getPageIdFromUrl() requires legacy routing. Configure "prepend_locale" or "url_suffix" in your config.yml file.');
```

Possible alternatives: "in your app configuration"

Reply:
Changed in a972f8d81da4a457cfd3dcadcb32c77e1d3c88aa